### PR TITLE
Explanation for how to benchmark Pulsar destinations

### PR DESCRIPTION
**Pull request by AndreasSko**
Added an explanation for how to use the [GalaxyBenchmarker](https://github.com/AndreasSko/Galaxy-Benchmarker) to benchmark Pulsar destinations.

### CODE REVIEW

File: source/benchmark.rst [hunk 1]
```diff
@@ -0,0 +1,104 @@
+Benchmarking your Pulsar setup
+==============================
+
+To determine the performance of your Pulsar setup, you can use the
+`GalaxyBenchmarker <https://github.com/AndreasSko/Galaxy-Benchmarker>`_.
```
Comment (bgruening) [inline source/benchmark.rst]: Would it be ok for you if we move this to usegalaxy-eu?

Comment (AndreasSko) [reply source/benchmark.rst]: Sure 👍

File: source/benchmark.rst [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+This is a framework that allows for easy benchmarking of various Galaxy
+job destinations.
+
```
Comment (gmauro) [inline source/benchmark.rst:8]: Could you add a prerequisites section describing the needs to have Docker Compose and pointing to Docker Compose overview (https://docs.docker.com/compose/)?

File: source/benchmark.rst [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+Configure the benchmarker
+-------------------------
+
+The benchmarker is configured using a yaml-configuration file. You can
+find a basic example in ``benchmark_config.yml.usegalaxyeu``. As a start,
+rename it to benchmark_config.yml and fill in the credentials for your
+regular UseGalaxy.eu user (a deeper explanation of the configuration
+can be found at https://github.com/AndreasSko/Galaxy-Benchmarker).
+To later benchmark your Pulsar setup, you will need an additional user
+for which UseGalaxy.eu is configured to only route jobs to your setup.
```
Comment (gmauro) [inline source/benchmark.rst]: This phrase is a bit unclear.

File: source/benchmark.rst [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+already defined.
+
+.. code-block:: yaml
+
+    destinations:
+      - name: YourDestinationName
+        type: Galaxy
+        galaxy_user_key: YOUR-USER-KEY-FOR-DESTINATION-USER
+
+Now, we just need to define the actual benchmark that we want to perform.
```
Comment (gmauro) [inline source/benchmark.rst:63]: Would it be useful to add a section describing the different Benchmarks types?

File: source/benchmark.rst [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+.. code-block:: yaml
+
+    workflows:
+      - name: YourCustomWorkflow
+        type: Galaxy
+        path: /src/custom-workflow-folder/custom-workflow.ga
+
+The benchmarker can submit jobs to multiple job destinations to compare
+the performance of each. As a start, we can stay with the one destination
+already defined.
```
Comment (gmauro) [inline source/benchmark.rst:54]: What do you mean? Is needed?

**Pull request merged**